features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/NewOverview/Overview.jsx

```diff
@@ -129,6 +132,11 @@ const styles = theme => ({
 function Overview(props) {
     const { classes, api: newApi } = props;
```

Comment:
DO we need classes `withStyles` props here ? I think we can use `useStyles` instead

Reply:
Let's do a code refactoring later, I just reorganized the existing code for layout changes.